exporter/omnitelk/hooks.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Omnition Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package omnitelk
+
+import (
+	"context"
+	"sync"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+)
+
+type tagCache struct {
+	sync.RWMutex
+	tags map[tag.Key]map[string]tag.Mutator
+}
+
+type kinesisHooks struct {
```

Comment:
```suggestion
type observabilityHooks struct {
```

or something else but there is nothing related to kinesis here.